tests/e2e/test_rosacli_upgrade.go

```diff
@@ -275,17 +276,24 @@ var _ = Describe("Cluster Upgrade testing",
 		It("to upgrade NON-STS rosa cluster across Y stream - [id:37499]", labels.Critical, labels.Runtime.Upgrade, func() {
 
 			By("Check the cluster version and compare with the profile to decide if skip this case")
+			if profile.Version != con.YStreamPreviousVersion || profile.ClusterConfig.STS {
+				Skip("Skip this case as the version defined in profile is not y-1 for non-sts cluster upgrading testing")
+			}
+
+			By("Check the cluster upgrade state to decide if skip this case")
```

Comment:
Let's check the available upgrade versions of the cluster in case there is a bug in rosacli